### DISCUSSION TITLE
iPhone 15 Pro 爆燙?!

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1579] [General]
不過，在iPhone 15 Pro熱銷之際，近來卻傳出過熱問題，更有網友戲稱iPhone 15 Pro是「火龍果」，使用相機4K 60P規格錄影約莫十五分鐘，就會出現發熱、卡頓等情況。
是因為台積電的晶片問題，還是因為使用鈦金屬散熱不佳?
https://tw.stock.yahoo.com/news/iphone-15-pro%E9%81%8E%E7%86%B1%E5%85%A8%E5%9B%A0%E5%8F%B0%E7%A9%8D%E9%9B%BB3%E5%A5%88%E7%B1%B3%EF%BC%9F-033906615.html

**Comment by uwje** [comment]
我認為近年來晶片製造廠商過度追求晶片的跑分效能，而有點過度的增加手機晶片的基礎頻率，導致近期不論是蘋果抑或是高通的晶片，皆出現容易過熱的情況，再加上近期講求AI的運算，如上方提到的高畫質錄影，便可能是通過數億次的運算獲得的，在相對緊湊的手機上運行高效能的晶片，本來就會出現散熱不佳的問題，所以除非將低運算較能、增加散熱空間、或提升製程使其在相同溫度下仍具相當的性能。
所以我認為出現發熱、卡頓等情況，可能是因為
1.晶片效能太高且散熱不足
2.追求手感，縮減散熱面積
3.製造工藝上產生問題，導致發熱量提升
4.晶片設計存在瑕疵
